### PR TITLE
Add ANSI and ISO layout for KBD67mkII soldered PCB

### PR DESCRIPTION
**Pull request by elmo-space**
## Description

Added the missing layouts for ANSI and ISO for the kbd67mkII soldered pcb.
Especially new guys in this hobby are often confused by the LAYOUT_all and don't know how to work with it so this should make creating keymaps easier for them.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by sigprof** [comment]
These layouts look exactly like the well-known `65_ansi_blocker` and `65_iso_blocker` layouts, so they should be named that way, and their names should be added to `rules.mk` to allow using the community layout keymaps with this PCB:

```
LAYOUTS = 65_ansi_blocker 65_iso_blocker
```

You may also consider adding `65_ansi_blocker_split_bs`.

**Comment by elmo-space** [comment]
> 
> 
> These layouts look exactly like the well-known `65_ansi_blocker` and `65_iso_blocker` layouts, so they should be named that way, and their names should be added to `rules.mk` to allow using the community layout keymaps with this PCB:
> 
> ```
> LAYOUTS = 65_ansi_blocker 65_iso_blocker
> ```
> 
> You may also consider adding `65_ansi_blocker_split_bs`.

good point! going to change that :)

File: keyboards/kbdfans/kbd67/mkii_soldered/info.json [hunk 1]
```diff
@@ -7,77 +7,289 @@
...
+                {"label":"R", "x":4.5, "y":1}, 
+                {"label":"T", "x":5.5, "y":1}, 
+                {"label":"Y", "x":6.5, "y":1}, 
+                {"label":"U", "x":7.5, "y":1}, 
+                {"label":"I", "x":8.5, "y":1}, 
+                {"label":"O", "x":9.5, "y":1}, 
+                {"label":"P", "x":10.5, "y":1}, 
+                {"label":"{", "x":11.5, "y":1}, 
+                {"label":"}", "x":12.5, "y":1}, 
+                {"label":"Enter", "x":13.75, "y":1, "w":1.25, "h":2},
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/mkii_soldered/info.json:181]: This needs to move...

```suggestion
```

File: keyboards/kbdfans/kbd67/mkii_soldered/info.json [hunk 2]
```diff
@@ -7,77 +7,289 @@
...
+                {"label":"F", "x":4.75, "y":2}, 
+                {"label":"G", "x":5.75, "y":2}, 
+                {"label":"H", "x":6.75, "y":2}, 
+                {"label":"J", "x":7.75, "y":2}, 
+                {"label":"K", "x":8.75, "y":2}, 
+                {"label":"L", "x":9.75, "y":2}, 
+                {"label":":", "x":10.75, "y":2}, 
+                {"label":"\"", "x":11.75, "y":2}, 
+                {"label":"~", "x":12.75, "y":2}, 
+                {"label":"PgUp", "x":15, "y":2},
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/mkii_soldered/info.json:196]: ... here.

```suggestion
                {"label":"Enter", "x":13.75, "y":1, "w":1.25, "h":2}, 
                {"label":"PgUp", "x":15, "y":2}, 
```

File: keyboards/kbdfans/kbd67/mkii_soldered/keymaps/iso/readme.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# The default keymap for KBD67 MKII
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/mkii_soldered/keymaps/iso/readme.md:1]:
```suggestion
# The default ISO keymap for KBD67 MKII
```

File: keyboards/kbdfans/kbd67/mkii_soldered/keymaps/ansi_split_bs/readme.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# The default keymap for KBD67 MKII
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/mkii_soldered/keymaps/ansi_split_bs/readme.md:1]:
```suggestion
# The default ANSI with Split Backspace keymap for KBD67 MKII
```

File: keyboards/kbdfans/kbd67/mkii_soldered/keymaps/ansi/readme.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# The default keymap for KBD67 MKII
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd67/mkii_soldered/keymaps/ansi/readme.md:1]:
```suggestion
# The default ANSI keymap for KBD67 MKII
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**